semgrep-core/tests/go/metavar_equality_stmt.TODO

```diff
@@ -0,0 +1,21 @@
+package main
```

Comment:
I shouldn't've deleted this file in my last PR - I didn't understand what the TODOs were yet.